test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -116,5 +116,52 @@ After the object
 "@
         $returnString | Should Be $expectedValue
     }
+
+    It "Test ConvertTo-HTML meta"{
+        $returnString = ($customObject | ConvertTo-HTML -Meta @{"author"="John Doe"}) -join $newLine
+        $expectedValue = normalizeLineEnds @"
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta name="author" content="John Doe">
+<title>HTML TABLE</title>
+</head><body>
+<table>
+<colgroup><col/><col/><col/></colgroup>
+<tr><th>Name</th><th>Age</th><th>Friends</th></tr>
+<tr><td>John Doe</td><td>42</td><td>System.Object[]</td></tr>
+</table>
+</body></html>
+"@
+        $returnString | Should Be $expectedValue
+    }
+
+    It "Test ConvertTo-HTML meta with invalid properties should throw warning" {
+        ($customObject | ConvertTo-HTML -Meta @{"authors"="John Doe";"keywords"="PowerShell,PSv6"} 3>&1) -match "Accepted meta properties are" | Should Be $true
```

Comment:
Since the warning message can be localized, you shouldn't check against the English string.  Instead, check that the warning message contains the meta pair.